<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyEditMesTypeRoom</class>
 <widget class="QDialog" name="lyEditMesTypeRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lyEditMesTypeRoom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DataManager.qrc">
    <normaloff>:/DataManger/Data/image/calendar.png</normaloff>:/DataManger/Data/image/calendar.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="meetSiteNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>现有会议地点名称</string>
      </property>
     </widget>
     <widget class="QComboBox" name="meetSiteComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="modifyMeetNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>需修改的会议地点</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modifyMeetSiteEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>添加新会议地点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>删除会议地点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改会议地点</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../DataManager.qrc"/>
 </resources>
 <connections/>
</ui>
